ranger/ext/img_display.py

```diff
@@ -805,6 +813,23 @@ def _late_init(self):
         self.pix_row, self.pix_col = x_px_tot // n_rows, y_px_tot // n_cols
         self.needs_late_init = False
 
+    def _handle_init_tmux(self):
+        # check if tmux allows passthrough
+        # use tmux escape sequence if it's enabled
+        # throw error if it's disabled
+        try:
+            tmux_allow_passthrough = check_output(
+                ['tmux', 'show', '-Apv', 'allow-passthrough']).strip()
```

Comment:
`tmux  show` is the shorthand for `tmux show-options`, I think it's better to use the long form because the meaning is more obvious..